style.css

```diff
@@ -31,16 +308,52 @@
 
 .home-grey-background {
   background-image: url("./images/home-background.jpg");
+  background-repeat: repeat-y;
+  height: 650px;
 }
```

Comment:
- [ ] Kindly add `background-size: cover;` to make your background image fits with all widths.

![image](https://user-images.githubusercontent.com/35535130/158815703-151e8787-56ff-4c03-b5bd-005b0d088fa8.png)